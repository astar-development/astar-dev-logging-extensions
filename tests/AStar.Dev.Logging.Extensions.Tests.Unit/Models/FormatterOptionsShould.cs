using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Logging.Extensions.Models;

[TestSubject(typeof(FormatterOptions))]
public class FormatterOptionsShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new FormatterOptions().ToJson().ShouldMatchApproved();
}