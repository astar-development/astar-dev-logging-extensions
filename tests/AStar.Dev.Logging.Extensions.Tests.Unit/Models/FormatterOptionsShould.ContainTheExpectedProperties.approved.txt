{
  "singleLine": false,
  "includeScopes": false,
  "timestampFormat": "HH:mm:ss ",
  "useUtcTimestamp": true,
  "jsonWriterOptions": {
    "indented": false
  }
}